### ISSUE TITLE
Left-hand panel: Carrot icons are reversed

### ISSUE THREAD

**Issue by HansVraga** [USGS-WiM/StreamStats#770]
See image
![image](https://user-images.githubusercontent.com/12058013/26998693-c8bc3352-4d55-11e7-884b-f278db7641c4.png)

**Comment by jknewson** [comment]
I think this is right. The V icon is to indicate that there is more information below, right? But I'm wondering if we just remove the > icon when it is selected. Does this make sense?

**Comment by HansVraga** [comment]
Really looks like we're following our own convention - but in other applications carrot down means open and carrot right means closed (example: https://test.wim.usgs.gov/SparrowGreatLakesV2/ )

https://ux.stackexchange.com/questions/37686/how-should-you-show-that-an-accordion-is-expanded

**Comment by jknewson** [comment]
according to the link you have posted, it looks as if there is no convention, Which, I've seen used multiple ways as well. Ours looks to follow the "Up and down chevrons on the right" convention. When we first implemented the accordion the >v arrows were meant to point the user to where they should be looking > go to the map to perform some steps, v look to the next step... type of idea. I can see this as being somewhat confusing now, because we have added additional steps in the accordion. However I'm still feeling the best approach would be to not show the > (or to reverse to the <) arrow when its selected and the v arrow to indicate there is content listed within the accordion. The v arrow will also indicate the natural progression through the sidebar steps as well.
